lms/djangoapps/grades/tasks.py

```diff
@@ -23,7 +23,9 @@
 
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
-def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher, raw_earned, raw_possible, **kwargs):
+def recalculate_subsection_grade(
+        user_id, course_id, usage_id, only_if_higher, weighted_earned, weighted_possible, **kwargs
+):
```

Comment:
Note that this task signature change will cause an error spike during deployment (old workers will still be alive and try to run tasks queued with the new signature's arguments); placing us on the devops "you done broke it" list. We'll need to speak with them ahead of rollout time to avoid this.

If we could stick the new `weighted_earned` and `weighted_possible` into kwargs, future changes could be easier.